src/Angor/Avalonia/Angor.Contexts.Funding/Founder/Operations/GetPendingInvestments.cs

```diff
@@ -0,0 +1,73 @@
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Angor.Contexts.Funding.Projects.Domain;
+using Angor.Shared;
+using Angor.Shared.Models;
+using Angor.Shared.Services;
+using CSharpFunctionalExtensions;
+using MediatR;
+
+namespace Angor.Contexts.Funding.Founder.Operations;
+
+public class GetPendingInvestments
```

Comment:
To be honest I am not crazy about multiple classes in one file, it means I can't see the logic and structure when I look at the solution files, how many of these IRequests do we have so far?